update structure and content of diagnostics section (#1076)
greenelab/covid19-review: content/10.diagnostics.md

<!-- To Do: possibly add text on what goes into developing new diagnostics?-->

The genetic sequence of the virus was first released by Chinese officials on January 10, 2020, and the first test to detect the virus was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
This information is important to the development of diagnostic approaches.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
As the pandemic has evolved throughout 2020 and 2021, a variety of tests have emerged within these two categories.

Molecular tests detect either viral RNA or protein in a patient sample.
Molecular tests are essential to identify infected individuals who may need treatment and whose contacts should quarantine.
Tests for viral RNA are done by reverse transcription (RT) of viral RNA to DNA followed by DNA amplification, usually by the polymerase chain reaction (PCR) [@DOI:10.1038/jid.2013.1].

> Tests for viral proteins are most often done using detection by an antibody pair in lateral flow tests (LFTs) [@URL:https://doi.org/10.1007/s00216-010-3661-4]. 

Reviewer: ```suggestion
Tests for viral proteins are most often done using detection by an antibody pair in lateral flow tests (LFTs) [@doi:10.1007/s00216-010-3661-4]. 
```

The low cost, excellent sensitity/specificity, and quick readout of RT-LAMP makes this an attractive alternative to RT-PCR. 
Alternative strategies like RT-LAMP are needed to bring widespread testing away from the lab and into under resourced areas.

One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity, and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6]. 
This test aims to bring the sensitivity of nucleic acid detection to the point of care or home testing setting. 
It could be applied for screening, diagnostics, or as a definitive test for people who are positive based on lateral flow tests.
This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
The authors break down the sensitivity of their test according to the cycle threshold (Ct) value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a Ct from RT-PCR of 32 or less. 

> The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

Reviewer: ```suggestion
The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).
However, there is some evidence suggesting that samples obtained from individuals that achieve Ct values >30 measured using RT-PCR tend to be less infective that those that record a Ct value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], and so RT-LAMP is still a useful diagnostic tool. 
```

> The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

Author: I didn't know that thanks - if there is a citation to back that up then I agree.

> The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

Reviewer: See what you think of my additional suggested change, of course edit as you see fit.

> The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

Reviewer: > @RLordan thanks for taking the time to go through my suggestions! I accepted most of them and replied to a couple. What should I do next?

Thanks @jessegmeyerlab, looks great! I just edited according to your feedback and commented back. I think sit tight until @rando2 reads through :)

> The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

Reviewer 2: Sorry for being out of the loop!

```suggestion
The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
```

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Reviewer: Also I think this particular kit is available from multiple places, the paper mentions thermofisher. Can you also clarify the number of primer sets used? the paper says 5?

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Author: Thanks, yes to all except the primer part - Amaral et al says they used primers from Zhang et al: "a primer set (typically composed of six primers)" and Zhang et al says "We designed 5 sets of LAMP primers targeting two fragments". If you look at the table in Zhang et al (@doi:10.1101/2020.02.26.20028373) it looks like five different sets of 6 were tested. So I think it is six primers? Thanks for catching.

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Author: What is the best way to change a suggestion? In this case I want to keep everything you did but change back to 6 primers. Do I need to reject it? I don't see a way to reject it.

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Reviewer: Yes, I see the distinction now. I changed back to six now.

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Reviewer 3: > What is the best way to change a suggestion? In this case I want to keep everything you did but change back to 6 primers. Do I need to reject it? I don't see a way to reject it.

You could click the "Resolve conversation" button, which effectively ignores the comment with the suggestion.  Then you can edit that part of the text separately to apply only the changes you want.  Or you could accept the proposed changes and then make another set of changes on top of them.  I don't know if you have permissions to edit the suggestion directly before committing.

> Various combinations of reagents, but an example is using WarmStart Colorimetric LAMP 2 × Master Mix (New England Biolabs) with a set of six primers developed previously by Zhang et al [@URL:https://doi.org/10.1101/2020.02.26.20028373]. 

Reviewer: I edited to six again... does that solve it? I think @jessegmeyerlab can accept it and it will be correct then.

To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed N-gene RNA standard (IVT RNA) were tested using quantities from 10^5 copies down to 10 copies. 

> The assay readout is the color of dye changing from red to yellow due to binding to the DNA product over 30 minutes. 

Reviewer: ```suggestion
The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes. 
```

RT-LAMP pilot studies for detecion of SARS-CoV-2 were recently reviewed and meta-analyzed [@PMID:34086539].
In the meta analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was 95.5% (CI 97.5% = 90.8-97.9%) and the cumulative specificity was 99.5% (CI 97.5% = 97.7-99.9%).
The low cost, excellent sensitity/specificity, and quick readout of RT-LAMP makes this an attractive alternative to RT-PCR. 
Alternative strategies like RT-LAMP are needed to bring widespread testing away from the lab and into under resourced areas.

One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity, and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6]. 
This test aims to bring the sensitivity of nucleic acid detection to the point of care or home testing setting. 
It could be applied for screening, diagnostics, or as a definitive test for people who are positive based on lateral flow tests.
This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.

> The authors break down the sensitivity of their test according to the cycle threshold (Ct) value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a Ct from RT-PCR of 32 or less. 

Reviewer 2: ```suggestion
The authors break down the sensitivity of their test according to the cycle threshold (C~t~) value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less. 
```

The performance is worse when considering all RT-PCR positive samples (including those with Ct values between 32-40).

> However, there is some evidence suggesting that samples obtained from individuals that achieve Ct values >30 measured using RT-PCR tend to be less infective that those that record a Ct value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], and so RT-LAMP is still a useful diagnostic tool. 

Reviewer 2: ```suggestion
However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool. 
```

##### CRISPR-based Detection

> Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) [@DOI:https://doi.org/10.1038/s41467-018-04252-2] has also been instrumental in scaling up testing protocols.

Reviewer: ```suggestion
Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) [@doi:10.1038/s41467-018-04252-2] has also been instrumental in scaling up testing protocols.
```

Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.

#### Protein tests
LFT-based tests are beneficial because they can detect current infection within 30 minutes and because they can be performed without specialized equipment at low cost. 

> LFTs rely on detection of viral protein with an antibody. 

Reviewer: ```suggestion
LFTs rely on the detection of viral protein with an antibody. 
```

> Often this is done with an antibody sandwich format, where one antibody containing a dye is binds at one site on the antigen, and an immobilized antibody on the strip binds at another site, allowing accumulation of the dye and the characteristic formation of a positive test line on the strip [@URL:https://doi.org/10.1007/s00216-010-3661-4].

Reviewer: ```suggestion
Often this is done with an antibody sandwich format, where one antibody containing a dye binds at one site on the antigen, and an immobilized antibody on the strip binds at another site, allowing accumulation of the dye and the characteristic formation of a positive test line on the strip [@doi:10.1007/s00216-010-3661-4].
```

> The applications of LFTs are broad: they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].

Reviewer: ```suggestion
The applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].
```

Current infection detection by LFTs is needed to enable the scale and speed of testing required to stop the spread of SARS-CoV-2.

> Lateral flow tests were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760].

Reviewer: ```suggestion
LFTs were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760].
```

Reviewer: I think multiple regions now or at least at a very low price in Europe

Author: Trying to find something citable that shows LFTs are free right now in the UK - is it possible to cite this website where they are ordered? https://www.nhs.uk/conditions/coronavirus-covid-19/testing/regular-rapid-coronavirus-tests-if-you-do-not-have-symptoms/

Author: The citation that is in there talks about how they are free last summer and that they might end them being free at the end of august.
